modules/apps/feature-flag/feature-flag-web/src/main/java/com/liferay/feature/flag/web/internal/feature/flag/FeatureFlagManagerImpl.java

```diff
@@ -44,6 +49,16 @@ public boolean isEnabled(long companyId, String key) {
 
 	@Override
 	public boolean isEnabled(String key) {
+		if (GetterUtil.getBoolean(
+				PropsUtil.get(
+					FeatureFlagConstants.getKey(
+						key,
+						ExtendedObjectClassDefinition.Scope.SYSTEM.
+							getValue())))) {
+
+			return isEnabled(CompanyConstants.SYSTEM, key);
+		}
+
```

Comment:
I think this logic should actually get moved inside of `isEnabled(long, String)`. Otherwise, we have a gap where the user could manually call that method with an invalid value.

Reply:
If we remove the block from `ConfigurationVisiblityUtil`, that's exactly what will happen.

Reply:
Done!